lk2nd/device/dts/msm8909/apq8009w-wtp.dts

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
```suggestion
// SPDX-License-Identifier: BSD-3-Clause
```

I'd prefer new submissions to be bsd3, which is also what's now preferred by linux for new dts.